AnkiDroid/src/main/assets/flashcard.css

```
@@ -124,33 +133,21 @@ preferred over using JavaScript.
+/* Keep the replay icon themeable while matching the historical triangle-only shape. */
 
-.replay-button span {
-  display: inline-block;
-  vertical-align: middle;
-  padding: 5px;
+.replay-button .play-action {
+  display: block;
+  width: 100%;
+  height: 100%;
+  color: inherit;
+  fill: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)